test/sub.mk

```diff
@@ -98,9 +98,9 @@ clean:
 ifeq ($(COMPILER_OUTPUT),ll)
 .SECONDEXPANSION:
 $(exe_files): $$(patsubst %.exe,%.bc,$$@) $$(filter $$(patsubst %.exe,%,$$@).%.bc, $(mod_bc_files)) $(RT)
-	$(CLANG) $(CFLAGS) -g -o $@ $^ 
+	$(CLANG) $(CFLAGS) -lm -g -o $@ $^ 
```

Comment:
`-lm` should be at the end. `-lm` isn't like a regular option: its order relative to `.o` files is significant.

Reply:
Here and below in one place

Reply:
fixed